index.html

```diff
@@ -52,7 +52,7 @@ <h4 class="connect font-style">LET'S CONNECT</h4>
     
                                 <li>
                                     <a class="social_medias" href="https://www.linkedin.com/in/kenbon-teshome/">
-                                        <img src="/images/Linkedin icon.svg" alt="Linkedin icon">
+                                        <img src="images/Linkedin icon.svg" alt="Linkedin icon">
                                     </a>
```

Comment:
- [ Optional] Your Linkedin icon is not displayed correctly kindly, replace the link with the following `<img src="./images/Linkedin icon.svg" alt="Linkedin icon">`